airbyte-integrations/connectors/source-mongodb-v2/integration_tests/expected_spec.json

```diff
@@ -177,6 +177,16 @@
         "order": 11,
         "group": "advanced"
       },
+      "initial_load_timeout_hours": {
+        "type": "integer",
+        "title": "Initial Load Timeout in Hours (Advanced)",
+        "description": "The amount of time an initial load is allowed to continue for before catching up on CDC logs.",
+        "default": 8,
+        "min": 4,
+        "max": 24,
+        "order": 10,
```

Comment:
I think order should be 12 here?

Reply:
fixed, now changed to 13, and append to the end...I think I now why... I didn't want to use the 13 number initially..